docs/dotnet/dotnet-signalr-metrics.md

```diff
@@ -22,22 +22,39 @@ This metric is required.
 <!-- semconv metric.signalr.server.connection.duration(metric_table) -->
 | Name     | Instrument Type | Unit (UCUM) | Description    |
 | -------- | --------------- | ----------- | -------------- |
-| `signalr_http_transport.connection.duration` | Histogram | `s` | The duration of connections on the server. [1] |
+| `signalr.server.connection.duration` | Histogram | `s` | The duration of connections on the server. [1] |
```

Comment:
the proposal here is to reflect that it's about HTTP in the attribute and not metric name